### ISSUE TITLE
Unable to restore tabs after computer crashed

### ISSUE THREAD

**Issue by emiljohansson** [brave/brave-browser#5550]
## Description 
My computer crashed and when I re-opened Brave, I selected the option to restore my tabs. When I selected it, nothing happened. I did try to use CMD+Shift+T to re-open them, but the list "Recently Closed" was empty.

I had a few windows, with a few tabs in each window.

## Steps to Reproduce

   1. Open the browser after a crash.
   2. Select to restore your tabs.

## Actual result:
No tabs were restored after selecting to restore them after the browser unexpectedly quit.

## Expected result:
Should restore all tabs after crash.

## Reproduces how often: 
Only happened once so far.

## Brave version (brave://version info)
0.66.101 Chromium: 75.0.3770.142 (Official Build) (64-bit)

## Version/Channel Information:
I use the current release.

## Other Additional Information:
I was able to restore all tabs in Chrome. This is not reproducible in Chrome - Version 75.0.3770.142 (Official Build) (64-bit)

Not sure if it matters, but the browser had an update available. I think the update icon was in the yellow state.

**Comment by poikilos** [comment]
- I had commented on a similar issue but couldn't find it. I've been having the issue too, about 1/30 times I run Brave. I usually have Brave open all day (1 session per day approximately). I finally got it to happen after enabling crash reports:
> Uploaded Crash Report ID d52be5be9a6e6362 (Local Context: Brave)
> Crash report uploaded on Sunday, August 4, 2019 at 8:15:50 AM
- But Brave uploads the report to chromium.org instead of brave: <https://bugs.chromium.org/p/chromium/issues/detail?id=990636#c1>
- ~/.config/chromium/Crash Reports/ is empty. Why?
